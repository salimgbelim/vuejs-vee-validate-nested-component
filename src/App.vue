<template>
  <div id="app" class="container">
    <!-- <div class="columns is-multiline">
       <simple-component></simple-component>
     </div>-->

    <MessageListV1 :messages="messages"></MessageListV1>
    <Form v-bind:reversed="true"></Form>
  </div>
</template>

<script>
  /* import SimpleComponent from './components/SimpleComponent.vue';
   import SimpleValidation from './components/vuelidate/SimpleValidation';


   export default {
     name: 'app',
     components: {
       SimpleComponent
     }
   }*/

  import MessageListV1 from './components/MessageListV1';
  import Form from './components/Form';

  export default {

    name: 'app',
    data: () => ({
      messages: ['Hey john', 'Oi john']
    }),
    components: {
      MessageListV1,
      Form
    }
  }

</script>

<style>
  #app {
    font-family: 'Avenir', Helvetica, Arial, sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
    text-align: center;
    color: #2c3e50;
    margin-top: 60px;
  }
</style>
