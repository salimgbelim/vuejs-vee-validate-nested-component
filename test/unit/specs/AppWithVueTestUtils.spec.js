import {shallow} from 'vue-test-utils';
import App from '@/App';

describe('App.test js with vue test utils and jest', () => {

  let cmp;

  beforeEach(() => {
    cmp = shallow(App, {
      data: {
        messages: ['Cat']
      }
    });
  });

  it('equals messages to ["Cat"]', () => {
    expect(cmp.vm.messages).toEqual(['Cat']);
  });

  it('has the expected html structure', () => {
    expect(cmp.element).toMatchSnapshot();
  });

});
