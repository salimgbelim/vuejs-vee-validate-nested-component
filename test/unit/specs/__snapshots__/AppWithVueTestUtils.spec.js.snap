// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.test js with vue test utils and jest has the expected html structure 1`] = `
<div
  class="container"
  id="app"
>
  <!---->
   
  <!---->
</div>
`;
